python/ray/tune/examples/FCNet_Trainable.py

```diff
@@ -0,0 +1,157 @@
+""" Trainable using FCNet Benchmark from Tabular Benchmarks for Hyperparameter Optimization and Neural Architecture Search """
```

Comment:
For all of these, can you have the same base class and just have 3 classes that inherit from that with different data URL class

also for this base class can you use `_setup` instead of __init__?